controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -236,7 +236,19 @@ http {
         ssl_verify_client on;
         ssl_verify_depth {{ $location.CertificateAuth.ValidationDepth }};
         {{ end }}
+        
+        {{ if (or $location.Redirect.ForceSSLRedirect (and (not (empty $server.SSLCertificate)) $location.Redirect.SSLRedirect)) }}
```

Comment:
@aledbf this seems broken. I think it should be moved back into the `location {{path}}` block.

What happens if we have one location with ssl-redirect: "false" and one with ssl-redirect: "true" - won't they now both be redirected?